src/app/components/list/overview/index.html

```diff
@@ -32,7 +32,7 @@
       defaultValue="SkyListInMemoryDataProvider"
       isOptional="true"
     >
-    Specifies a data provider to obtain the data to display. The list component requires this property or the <sky-code>data</sky-code> property. Acceptable values: <sky-code>extends ListDataProvider</sky-code>. To extend ListDataProvider, provide a <sky-code>get</sky-code> function that accepts a <sky-code>ListDataRequestModel</sky-code> as an argument and returns an <sky-code>Observable ListDataResponseModel </sky-code>, and a <sky-code>count</sky-code> function that returns an <sky-code>Observable number</sky-code> that indicates how many total records are in the list.
+    Specifies a data provider to obtain the data to display. The list component requires this property or the <sky-code>data</sky-code> property. Acceptable values: <sky-code>extends ListDataProvider</sky-code>. To extend ListDataProvider, provide a <sky-code>get</sky-code> function that accepts a <sky-code>ListDataRequestModel</sky-code> as an argument and returns an <sky-code>Observable ListDataResponseModel </sky-code>, and a <sky-code>count</sky-code> function that returns an <sky-code>Observable number</sky-code> that indicates how many total records are in the list. You must control the filtering and searching when you opt in to the <sky-code>dataProvider</sky-code>.
```

Comment:
A few things...

Is this new sentence strong enough? I mean, what do we mean by "you must control"? Does that adequately indicate that the user (if it is in fact the user) is taking responsibility? I wonder if it would be better to say something more like "For lists that use `dataProvider` instead of `data`, the consumer must take responsibility for filtering and searching." Not really sure; that's just my initial question after looking at this and the first bullet of the description at https://github.com/blackbaud/skyux2-docs/issues/466.

And speaking of https://github.com/blackbaud/skyux2-docs/issues/466, this seems to correlate to the first bullet in that description. Do we need to include something for the second bullet as well?

And do we need to create a separate issue for updating the demo?